config-gui.py

```diff
@@ -147,27 +147,30 @@ def see_registry_internal(self) -> None:
 (Error: '{str(exc)}')""")
             return None
         # run directly the SQL command to get the storage
-        CUR.execute("SELECT vendor_id, product_id, odometer_read, cost, datetime FROM Prompt WHERE datetime >= ( ? ) AND datetime <= ( ? )", ( date_a, date_b ))
+        CUR.execute("SELECT vendor_id, product_id, odometer_read, cost, datetime, noentry_reason FROM Prompt WHERE datetime >= ( ? ) AND datetime <= ( ? )", ( date_a, date_b ))
         finale = CUR.fetchall()
         # analyze and translate
         finale_str = "\n"
         products_operator = {}
         vendors_operator = {}
         final_cost = 0
         final_odometer = 0
-        for v, p, o, c, d in finale:
+        for v, p, o, c, d, r in finale:
             # translate some IDs
             CUR.execute("SELECT name FROM Vendors WHERE id == ( ? )", ( v, ))
             v = CUR.fetchone()[0]
             CUR.execute("SELECT name FROM Products WHERE id == ( ? )", ( p, ))
             p = CUR.fetchone()[0]
             # map some of the data
+            noentry_msg = ""
+            if r != "N/A":
+                noentry_msg = f"ATENCION: {r}"

```

Comment:
So, the warning won't be shown if the reason is `N/A`. Can the user override the warnings to avoid a reason?

Reply:
I mean, if the reason is set to `N/A` by the user?

Reply:
If that happens, the warning will be ignored. But this could be reasonable: `N/A` does not mean anything, and maybe we can patch this later.

But this is not user-facing thing...